### PR TITLE
Fix longitudinal report

### LINKED ISSUE #5761: Test report broken
Our test report job is again failing

### PR DESCRIPTION
**Pull request by ian-r-rose**
There was some weird variation in behavior here depending on precise versions of libraries that I didn't really track down, but the heart of the issue was that, a few days ago, a test failure message included a full HTML string that was not escaped, and messed up the resulting HTML document.

Fix is live on my [fork](https://ian-r-rose.github.io/distributed/test_report.html)

- [x] Closes #5761
- [ ] Tests added / passed
- [x] Passes `pre-commit run --all-files`

### CODE REVIEW

File: continuous_integration/scripts/test_report.py [hunk 1]
```diff
@@ -172,7 +173,7 @@ def dedup(group):
         for w in workflows
         if (
             pandas.to_datetime(w["created_at"])
-            > pandas.Timestamp.now(tz="UTC") - pandas.Timedelta(days=90)
+            > pandas.Timestamp.now(tz="UTC") - pandas.Timedelta(days=60)
```
Comment (fjetter) [inline continuous_integration/scripts/test_report.py:176]: why reduce this?

**Review by fjetter — COMMENTED** [review]
> a test failure message included a full HTML string that was not escaped, and messed up the resulting HTML document.

I know we made the entire report more robust, primarily for missing artifacts. Is there a way to spot these kind of errors?

**Comment by ian-r-rose** [comment]
> I know we made the entire report more robust, primarily for missing artifacts. Is there a way to spot these kind of errors?

I don't think there is an easy way -- the test failure in question was checking for a specific string in the HTML for the dask dashboard.

To be clear: this fix *shouldn't* be necessary. Vega specs should be robust to unusual strings, and indeed, this probably represents a security hole in Vega where a malicious user could inject HTML/JS to do something bad to a user. I suspect a bug was recently introduced in the Altair+Vega+VegaLite stack somewhere, but I wasn't able to find it in a timely fashion, and I wanted to unblock this visualization.

File: continuous_integration/scripts/test_report.py (hunk 1 above)
Comment (ian-r-rose) [reply continuous_integration/scripts/test_report.py:176]: I'm starting to be concerned about bumping up against GitHub's rate limit for the use of `GITHUB_TOKEN` in actions (currently 1000 requests per hour). Roughly speaking, we are downloading ~10 artifacts per workflow run, plus a slew of administrative requests here. An alternative would be to just choose the most recent ~60-80 runs or thereabouts.

Comment (fjetter) [reply continuous_integration/scripts/test_report.py:176]: ok, got it. 

I think what we're looking for is indeed "last X runs" and not "last X days". The way I understand this code, that would be very easy to change, wouldn't it?

**Review by fjetter — APPROVED** [review]
LGTM. If you're up for it you may change the Xdays to Xruns but I'm fine with merging this either way

**Comment by fjetter** [comment]
As always, thanks @ian-r-rose for fixing these builds so quickly

**Comment by ian-r-rose** [comment]
> I think what we're looking for is indeed "last X runs" and not "last X days". The way I understand this code, that would be very easy to change, wouldn't it?

That's right -- I'll push a change for this right now

**Comment by ian-r-rose** [comment]
Done

**Pull request merged**